examples/fillform.js

```diff
@@ -7,36 +7,35 @@ export const options = {
   }
 }
 
-export default function() {
+export default async function() {
   const browser = chromium.launch({
     headless: __ENV.XK6_HEADLESS ? true : false,
   });
   const context = browser.newContext();
   const page = context.newPage();
 
-  // Goto front page, find login link and click it
-  page.goto('https://test.k6.io/', { waitUntil: 'networkidle' }).then(() => {
-    return Promise.all([
+  try {
+    // Goto front page, find login link and click it
+    await page.goto('https://test.k6.io/', { waitUntil: 'networkidle' });
+    await Promise.all([
```

Comment:
@inancgumus Just for my own understanding, can we also do?
```
await page.waitForNavigation()
await page.locator('a[href="/my_messages.php"]').click()
```

Reply:
Yeah, it should work, but `goto` and `waitForNavigation` have different implementations beneath. Btw, `locator.click` is not yet async.